hugo/static/.htaccess

```diff
@@ -42,6 +43,9 @@ RewriteRule ^[A-Za-z]/[A-Za-z][0-9][0-9]/([A-Za-z])([0-9][0-9])\-([0-9][0-9][0-9
 # This way there is only one page. Should maintain for backward compatibility for some time after August 2019.
 RewriteRule ^papers/[A-Za-z]/[A-Za-z][0-9][0-9]/([A-Za-z])([0-9][0-9])\-([0-9][0-9][0-9][0-9])/?$ https://www.aclweb.org/anthology/$1$2-$3/ [R=301,L]
 
+# Volume pages (e.g., redirect /W19-75 to /volumes/W19-75)
+RewriteRule ^([A-Za-z])([0-9][0-9])\-([0-9][0-9])$ https://www.aclweb.org/anthology/volumes/$1$2-$3/ [R=301,L]
```

Comment:
Should this also allow a slash at the end, i.e. end the matching with `/?$`? Would not hurt in my opinion.

Reply:
Yes, good catch, pushed.